src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1128,6 +1128,18 @@ void TwitchMessageBuilder::appendTwitchBadges()
                 continue;
             }
         }
+        else if (badge.key_ == "vip")
+        {
+            if (auto customVipBadge = this->twitchChannel->ffzCustomVipBadge())
+            {
+                this->emplace<ModBadgeElement>(
```

Comment:
a bit pedantic: `ModBadgeElement` just feel weird to me here after all this is handling VIP badges.

Reply:
Yes, I am aware of this - I will rework all references to custom mod badges and rename those to something along "authority badge" (since both badges use the same channel authority slot). This pull request is basically a pure copyasta, so I think it's better to have less duplicated code by reworking already existing stuff. This is also why the PR is a draft for now. I'll fix it tommorow because I'm tired now.

Reply:
Fixed in a23eb20